tests/apps/perf/tester/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN go build -o tester .
 FROM golang:1.15 as fortio_build_env
 
 WORKDIR /fortio
-RUN git clone https://github.com/artursouza/fortio.git #redo
+ADD "https://api.github.com/repos/artursouza/fortio/branches/master" skipcache
```

Comment:
do we need forked version of fortio ?

Reply:
Yes, this is to have uuid dynamically generated in the URL. I have a PR to merge into their repo but that work will take longer. This is key to avoid having one more http layer in the perf test: fortio -> http proxy to handle test call and inject uuid -> daprd

Reply:
https://github.com/fortio/fortio/pull/426